boms/minimal/pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.oasp.java.dev</groupId>
+    <artifactId>oasp4j</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
```

Comment:
- Currently points to oasp4j as parent. Should be oasp4j-boms instead as per the package hierarchy hence the parent is not getting resolved correctly and resulting in pom as non-parseable.
- As a suggestion for file name. If this BOM is to define only oasp4j modules it can be named as oasp4j-modules-bom, which I find to be more descriptive.

Reply:
Wow! I think you also pointed me to the reason of the travis errors. Still it is kind of magic why that can not be reproduced locally even after purging my local m2 repo but I will give it a try to also remove the install hack in `.travis.yml`.